CMakeLists.txt

```diff
@@ -224,6 +224,12 @@ if (JEMALLOC_FOUND)
   set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -ljemalloc -L${JEMALLOC_LIBRARY_PATH}")
 endif()
 
+#---Here we add liburing to the linker flags if needed------------------------------------------
+if (LIBURING_FOUND)
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -luring -L${LIBURING_LIBRARY_PATH}")
```

Comment:
Isn't it more common to have `-Lpath -llibrary` instead of the other way around? After all, `-L` adds to the search path needed to find the `-l`'ed library, also in this case, IIUC?